main.nf

```diff
@@ -571,6 +581,19 @@ workflow {
             CAT_DB.out.db
         )
         ch_software_versions = ch_software_versions.mix(CAT.out.version.first().ifEmpty(null))
+
+        /*
+         * GTDB-tk: taxonomic classifications using GTDB reference
+         */
+        if ( params.gtdb ){
+            // TODO apply only to medium & high quality MAGs (>=50% complete, <=10% contamination)?
```

Comment:
If the GTDB database contains bacterial and archaeal references, than this would be currently not a good idea, because:
- BUSCO requires a database that is e.g. archaea or bacteria specific. When choosing the bacteria db, all values for archaea genomes will be strongly underestimated (e.g. a 100% complete archaea genome will be reported as 20-40% complete...).
- some microorganism with reduced genomes (e.g. symbiontic, pathogenic,...) might have with general BUSCOs low completeness estimates but are fine. So I would not fancy forward flaws with gene sets used to evaluate with BUSCO downstream.

Reply:
-> use Busco auto lineage function